nemo_curator/scripts/aegis_classifier_inference.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Similar to my other comment, maybe `aegis_classifier_inference.py` and the others can be moved into their own subdirectory here too.